CMakeLists.txt

```diff
@@ -0,0 +1,54 @@
+cmake_minimum_required(VERSION 3.16)
```

Comment:
This requirement is unnecessarily high. There is no feature used in the CMake scripts that require this version. A header-only project can very comfortably use 3.14 at the lowest.

Reply:
You're right I took this from another one.... thought FetchContent was quite high but apparently its only 11.

Reply:
`MakeAvailable` is available since 3.14, but it's really only a convenience function wrapping `Populate` and `add_subdirectory`. I would say you should stick to 3.14 regardless, because it has better support for header-only libraries (`DESTINATION` is not mandatory in target install rules, `ARCH_INDEPENDENT`).